qpc/cred/conftest.py

```diff
@@ -20,7 +20,7 @@ def _setup_server_config_file():
     Since all cli commands require qpc config and qpc login to be executed,
     require_token must be False, to avoid passing login info
     """
-    from qpc.utils import write_server_config
+    from qpc.utils import write_server_config  # pylint: disable=import-outside-toplevel
```

Comment:
because this type of import will be common on conftest.py you may put disable this lint for the whole file